boards/arm/nrf52840_blip/Kconfig.defconfig

```diff
@@ -29,4 +29,25 @@ endif # IEEE802154
 config BT_CTLR
 	default BT
 
+if PWM
+
+config PWM_0
+	default y
```

Comment:
For all those you can use

`default <whatever you have in your if statement>`

so:

`default PWM`
`default I2C`
etc